WeScan/ImageScannerController.swift

```diff
@@ -136,7 +148,32 @@ public final class ImageScannerController: UINavigationController {
             view.trailingAnchor.constraint(equalTo: blackFlashView.trailingAnchor)
         ]
         
-        NSLayoutConstraint.activate(blackFlashViewConstraints)
+        var constraints = blackFlashViewConstraints
+        
+        if (self.view.contains(selectPhotoButton)) {
+            var selectPhotoButtonConstraints = [NSLayoutConstraint]()
+            
+            
+            if #available(iOS 11.0, *) {
+                selectPhotoButtonConstraints = [
+                    selectPhotoButton.widthAnchor.constraint(equalToConstant: 44.0),
+                    selectPhotoButton.heightAnchor.constraint(equalToConstant: 44.0),
+                    selectPhotoButton.rightAnchor.constraint(equalTo: view.safeAreaLayoutGuide.rightAnchor, constant: -24.0),
+                    view.safeAreaLayoutGuide.bottomAnchor.constraint(equalTo: selectPhotoButton.bottomAnchor, constant: (65.0 / 2) - 10.0)
+                ]
+            } else {
+                selectPhotoButtonConstraints = [
+                    selectPhotoButton.widthAnchor.constraint(equalToConstant: 44.0),
+                    selectPhotoButton.heightAnchor.constraint(equalToConstant: 44.0),
+                    selectPhotoButton.rightAnchor.constraint(equalTo: view.rightAnchor, constant: -24.0),
+                    view.bottomAnchor.constraint(equalTo: selectPhotoButton.bottomAnchor, constant: (65.0 / 2) - 10.0)
+                ]
+            }
+            
+            constraints = constraints + selectPhotoButtonConstraints;
+        }
+        
+        NSLayoutConstraint.activate(constraints)
     }
     
```

Comment:
<!--
  0 failure: 
  1 warning:  Lines should not ...
  
  
  DangerID: danger-id-Danger;
  File: WeScan/ImageScannerController.swift;
  Line: 178;
-->

- :warning: Lines should not have trailing whitespace. (`trailing_whitespace`)